PubSub/tests/System/PublishAndPullTest.php

```diff
@@ -90,29 +90,40 @@ public function testPublishMessagesAndPull($client)
      */
     public function testOrderingKeys($client)
     {
-        if ($client instanceof PubSubClientRest) {
-            $this->markTestSkipped(
-                'enableMessageOrdering not available in REST transport during experimental period.'
-            );
-        }
-
         list ($topic, $sub) = self::topicAndSubscription($client, [], [
             'enableMessageOrdering' => true
         ]);
 
         $key = 'foo';
+        $numOfmessages = 5;
```

Comment:
```suggestion
        $numMessages = 5;
```

Reply:
@vishwarajanand I see the snake case is not followed in the PR, How about changing it to `numOfMessages`? because the test is not counting message with this variable, instead, stating this is the number of messages the test is planning to send. What do you think?

Reply:
Done